public/js/app.js

```diff
@@ -1,52 +1,72 @@
-angular.module('mean', ['ngCookies', 'ngResource', 'ui.bootstrap', 'ui.route', 'mean.system', 'mean.directives'])
-  .config(['$routeProvider',
-      function($routeProvider) {
-          $routeProvider.
-          when('/', {
-            templateUrl: 'views/index.html'
-          }).
-          when('/app', {
-            templateUrl: '/views/app.html',
-          }).
-          when('/privacy', {
-            templateUrl: '/views/privacy.html',
-          }).
-          when('/bottom', {
-            templateUrl: '/views/bottom.html'
-          }).
-          when('/signin', {
-            templateUrl: '/views/signin.html'
-          }).
-          when('/signup', {
-            templateUrl: '/views/signup.html'
-          }).
-          when('/choose-avatar', {
-            templateUrl: '/views/choose-avatar.html'
-          }).
-          otherwise({
-            redirectTo: '/'
-          });
-      }
-  ]).config(['$locationProvider',
-    function($locationProvider) {
-        $locationProvider.hashPrefix("!");
+/* eslint-disable */
```

Comment:
This is an inappropriate way of handling the violations _eslint_ flags for you. I suggest you do what is appropriate on this file by resolving the violations..

Reply:
Ok, I will look into it

Reply:
Ok I am suppose to use /* eslint no-undef: "error" */ instead  of disabling eslint for the whole file. Thanks